### PR TITLE
Setup Application

### PR DESCRIPTION
**Pull request by maen-al-khraisha**
This pull request will be for installing the main structure of the application.

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @maenkhraisha 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

#### You have submitted a project that the pull request summary is not descriptive enough and the README.md file is not professional. For the README.md file, kindly use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) as a guide and include instructions for cloning the project, installing all dependencies, building the dist folder and running the server locally. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: src/Components/AddBook.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const AddBook = () => (
+  <div>
+    <p>ADD NEW BOOK</p>
+    <input type="text" placeholder="Book Title" />
+    <select>
+      {categoriesList.map((category) => (
+        <option key={category.key} value={category.title}>{category.title}</option>
+      ))}
+    </select>
+  </div>
```
Comment (rustam-tolipov) [inline src/Components/AddBook.js:18]: - [ ] It seems like you forgot to add `ADD BUTTON` to your AddBook.js. Kindly add your button in here.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,73 @@
...
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+## Acknowledgments
 
-### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
 
-### `npm run build` fails to minify
+## 📝 License
```
Comment (rustam-tolipov) [inline README.md:71]: - [ ] [OPTIONAL] Kindly implement your MIT license as well.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @maenkhraisha, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @maenkhraisha, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**